cypress/support/step_definitions/dictionary/addBulkConceptsToDictionary/addBulkConceptsToDictionary.ts

```diff
@@ -0,0 +1,128 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import { 
+  After, 
+  Before, 
+  Given, 
+  Then, 
+  When } from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser, getConceptId } from '../../../utils';
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+  cy.url().should(
+    "contain",
+    `/users/${getUser()}/collections/${getDictionaryId()}/concepts/`
+  );
+});
+
+Given('the user is on the "Add concepts in bulk from CIEL" page', () => {
+cy.visit(
+`/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+);
+cy.url().should(
+"contain",
+`/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+  );
+});
+
+Given('CIEL concept "1000" is already in the dictionary', () => {
+  cy.visit('/actions/');
+  cy.url().should(
+    "contain", '/actions/'
+  );
+});
+
+When('the user clicks the "Add concepts" button', () => {
+  cy.findByText("Add concepts").click({force: true});
```

Comment:
Error from cypress:

`Unable to find an element with the text: Add concepts. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.`

Reply:
I wouldn't use `{force: true}` without very good reason. If the button is disabled and we're forcing the click on it, we're doing something the end-user can't. If the button isn't enabled yet, we need to figure out why.

Reply:
@ibacher the explanation makes sense to me. Thanks so so much!